### ISSUE TITLE
Suggestion: allow use of single quote in nested translations.

### ISSUE THREAD

**Issue by JoshMcCullough** [i18next/i18next#695]
Is it possible, when parsing nested translations, to convert single quotes to double quotes? This would allow for less verbose, easier-to-read translations while maintaining JSON compatibility.

E.g., instead of...

```
{ "girlsAndBoys_plural": "$t(girls, {\"count\": {{girls}} }) and {{count}} boys" }
```

we could do...

```
{ "girlsAndBoys_plural": "$t(girls, {'count': {{girls}} }) and {{count}} boys" }
```

This is a simple example, but complex examples would benefit must more from this change.

**Comment by jamuhl** [comment]
should be possible...just need to replace that before interpolating: https://github.com/i18next/i18next/blob/master/src/Interpolator.js#L88

if you like you can provide a PR or i will see if i can do it next week - on holiday right now

**Comment by jamuhl** [comment]
published in i18next@4.1.0
